flask_backend/routes/movie.py

```diff
@@ -39,45 +39,43 @@ def posters_cubism():
 
 @bp.route("/movies/posters/images")
 def poster_images():
-    lazy_loading = request.headers.get("X-LAZY-LOAD", None)
-    if lazy_loading is None or lazy_loading != "1":
+    lazy_loading = request.headers.get("X-LAZY-LOAD", "0")
+    if lazy_loading != "1":
         abort(400)
-    page = request.args.get("page", 0)
-    limit = 4
+
     try:
-        page = int(page)
+        page = int(request.args.get("page", 1))
+        limit = int(request.args.get("limit", 4))
     except ValueError:
         abort(400)
 
     user_logged_in = g.user is not None
-    movies = get_paginated(page, limit, user_logged_in)
+    scrennigs_list = get_paginated(page, limit, user_logged_in)
```

Comment:
aqui tem um pequeno typo, acho que é `screenings_list`:

```suggestion
    scrennnigs_list = get_paginated(page, limit, user_logged_in)
```

(daí tem que mudar ali embaixo nos outros lugares onde é utilizado).

Reply:
Ajustei junto com o merge